### ISSUE TITLE
Managed Kiam

### ISSUE THREAD

**Issue by pipo02mix** [giantswarm/roadmap#130]
**Is your feature request related to a problem? Please describe.**

AWS customers usually get profit of Kiam to control access to AWS API. Our AWS tenant clusters already contain [this component](https://github.com/giantswarm/kiam-app).

**Describe the solution you'd like**

Include this component as Managed App in our UI, consider allow the customer to configure any other parameter and polish the documentation (README)

**Describe alternatives you've considered**

Move to [IAM roles for services accounts](https://github.com/giantswarm/roadmap/issues/114) and remove kiam

**Comment by cokiengchiara** [comment]
thanks @pipo02mix ! 

You seem to mean sth like "Add Kiam as an app in Happa", right?

You don't mean "Managed Kiam," as in _a managed apps team should take care of this for customers with SLA_?

**Comment by pipo02mix** [comment]
No really, I created a ticket [here](https://github.com/giantswarm/roadmap/issues/129) to show the App in Happa (which I think @oponder has already solved :) and can be closed).

This is more about opening the Kiam App we used for our own components, to be used for the customers too as Managed App. 

This one is important because there will be problematic to run two Kiam Apps in the same cluster (one for us and one for the customer), due to the way it works. Customers expect to use our Kiam App when they move to new GS version
